python/lsst/ts/watcher/watcher_csc.py

```diff
@@ -139,48 +146,50 @@ async def escalate_alarm(self, alarm):
             * alarm.escalated_id is not blank: the alarm was already
               escalated (or at least an attempt was made).
             * alarm.do_escalate false: alarm should not be escalated.
-            * alarm.escalation_responders empty: there is nobody to escalate
+            * alarm.escalation_responder empty: there is nobody to escalate
               the alarm to (so do_escalate should never have been set).
         """
         if alarm.escalated_id:
             raise RuntimeError("Alarm already escalated")
         if not alarm.do_escalate:
             raise RuntimeError("Alarm do_escalate false")
-        if not alarm.escalation_responders:
-            raise RuntimeError("Alarm escalation_responders empty")
+        if not alarm.escalation_responder:
```

Comment:
Does `not` work correctly on an empty string as well? Because I seem to have understood that that may get passed on and not `None`.

Reply:
An empty string is treated as logically false. And this handles None as well as "" (I don't remember if None is an allowed value, but I can imagine that in a future code update it might be one).